superset/assets/javascripts/explorev2/reducers/exploreReducer.js

```diff
@@ -90,6 +87,7 @@ export const exploreReducer = function (state, action) {
           chartUpdateEndTime: null,
           chartUpdateStartTime: now(),
           triggerQuery: false,
+          queryRequest: action.queryRequest,
```

Comment:
@vera-liu I cleaned up `mapStateToFields` from the fields' state a few commits ago for that reason, but I guess we can remove the request object from the state later if it creates an issue. For instance if we were to persist the state to local storage it might not work, but we're not planning on doing that so it's ok.